doc/bn.tex

```diff
@@ -1906,9 +1906,9 @@ \section{Combined Modular Reduction}
 
 \chapter{Exponentiation}
 \section{Single Digit Exponentiation}
-\index{mp\_expt\_u32}
+\index{mp\_expt\_n}
 \begin{alltt}
-mp_err mp_expt_u32 (const mp_int *a, uint32_t b, mp_int *c)
+mp_err mp_expt_n(const mp_int *a, int b, int *c)
```

Comment:
Signature of mp_expt_n in bn.tex doesn't match the signature in header-file and implementation